src/components/Dashboard/TodoList.vue

```diff
@@ -1196,6 +1199,7 @@ export default class TodoList extends Mixins(
       const payErrorObj = paymentStatusCode && await this.getPayErrorObj(paymentStatusCode)
 
       const draft = changeOfRegistration
+      // FUTURE: verify that this checks for all possible data:
```

Comment:
There may be outstanding work here to identify if any changes were made. This is way out of scope for ticket 11954, but if you notice something while testing (or you are in this code in the future), perhaps verify this (and also for conversion filings below).